src/components/pages/room.js

```diff
@@ -1,47 +1,99 @@
-import React from 'react';
 import PropTypes from 'prop-types';
-import { Row, Space, List } from 'antd';
+import React, { useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { useDateFormat } from '../language-context.js';
-import { roomShape } from '../../ui/default-prop-types.js';
+import { Row, Space, List, Collapse, Button } from 'antd';
+import { invitationShape, roomShape } from '../../ui/default-prop-types.js';
+import RoomInvitationCreationModal from '../room-invitation-creation-modal.js';
 
 export default function Room({ PageTemplate, initialState }) {
   const { t } = useTranslation('room');
   const { formatDate } = useDateFormat();
-  const { roomDetails } = initialState;
+  const { room, invitations } = initialState;
+  const [isRoomInvitationModalOpen, setIsRoomInvitationModalOpen] = useState(false);
+
+  const handleOpenInvitationModalClick = event => {
+    setIsRoomInvitationModalOpen(true);
+    event.stopPropagation();
```

Comment:
I think this might not be needed here, right? 🤔

Reply:
It is, otherwise the collapse also collapses when you click the button.

Reply:
Got it 👍